ocaml-lsp-server/docs/ocamllsp/construct-spec.md

```diff
@@ -0,0 +1,44 @@
+# Construct Request
+
+## Description
+
+Provides commands to browse and fill typed holes (`_`). Such holes sometimes
+appear in the result of other commands like `destruct` and can also be inserted
+manually in the source. The command is already accessible via a completion hook,
+however, in certain situations, invoking `construct` on a hole via a request
+allows more control.
+
+## Client Capability
+
+There is no client capability relative to this request.
+
+## Server capability
+
+- property name: `handleConstruct`
+- property type: `boolean`
+
+## Request
+
+- method: `ocamllsp/typeEnclosing`
```

Comment:
I assume typeEnclosing is a typo

Reply:
Fixed on https://github.com/ocaml/ocaml-lsp/pull/1348/commits/b2939c252136a00bd02e561a6275dd5f30ee1756

Nice catch, thanks!